### ISSUE TITLE
[Request] Bring back functionality to quickly open Breakpoint and Scripting windows from main window

### ISSUE THREAD

**Issue by crankygeek** [ProxymanApp/Proxyman#982]
### Proxyman version? (Ex. Proxyman 1.4.3)
Version 2.32.1 (23210)

### macOS Version? (Ex. mac 10.14)
MacBook Pro (16-inch, 2019) macOS 11.5.2 (Intel)

I'd swear that Scripting and Breakpoint used to have indicators in the status bar that would open the related window when clicked on. Do I remember that correctly? Currently, unless you have the toolbar buttons configured, there's no indication in the main window if either of these functions are turned on. Also, I think the UI in the status bar is a bit more clear to indicate 
when these are turned on, vs. relying on noticing the color state of the toolbar buttons.

Proxy Override, Network Throttling and No Caching have status bar indicators. Can we get the label and window open functionality for Scripting and Breakpoint too?

**Comment by crankygeek** [comment]
Had another thought about quick-opening those screens. Maybe add Option+click on the toolbar buttons to open them? Though I'm not sure about discoverability, or how that would align with the HIG.

**Comment by NghiaTranUIT** [comment]
> I'd swear that Scripting and Breakpoint used to have indicators in the status bar that would open the related window when clicked on.

Yes, it's true, but the tool must be enabled and have one active rule at least. Then the badge at the bottom will appear and you can click on it to open it.

I intentionally design this way because if we enable all tools (by default, these tools are enabled), all tools will appear on the bottom bar -> Look messy. 

> Proxy Override, Network Throttling and No Caching have status bar indicators. Can we get the label and window open functionality for Scripting and Breakpoint too?

It's a reasonable idea. Let see if I can add it.

To workaround, you might use a shortcut to quickly open the tool 👍

**Comment by crankygeek** [comment]
I'm old and can't remember all the keyboard shortcuts available across all of my apps. 🤣 But point taken.
